### PR TITLE
Changes job states from squares colors to icons

### PR DESCRIPTION
**Pull request by qhuynh96**
* Also fixes the fail jobs notice jumped above the status
a3jgu1wmvxb

### CODE REVIEW

File: prow/cmd/deck/static/user-script.js [hunk 1]
```diff
@@ -172,15 +172,30 @@ function createPRCardTitle(pr, tidePools, jobStatus, mergeAbility) {
...
+    const getStateIcon = (state) => {
+        switch (state) {
+            case "success":
+                return "check_circle";
+            case "failure":
+                return "error";
+            case "pending":
+                return "watch_later";
+            case "triggered":
+                return "access_time";
```
Comment (BenTheElder) [inline prow/cmd/deck/static/user-script.js]: I think this is already out of sync with the other script

Comment (qhuynh96) [reply prow/cmd/deck/static/user-script.js]: done

**Comment by BenTheElder** [comment]
Maybe toss up a screenshot of what it looks like after this?

**Comment by qhuynh96** [comment]
@BenTheElder put the screenshot in

**Comment by qhuynh96** [comment]
@BenTheElder attach the screenshot

**Comment by BenTheElder** [comment]
we should consider reducing the right padding on the checkmarks etc on the overall status line and on each of the jobs, especially on mobile it seems ~2x necessary

**Comment by BenTheElder** [comment]
Icons look great though, thanks!

**Comment by stevekuznetsov** [comment]
Icons look good from here!

/hold cancel

**Pull request merged**